modules/simple_forms_api/app/services/simple_forms_api/form_upload_notification_email.rb

```diff
@@ -20,7 +20,7 @@ def initialize(config, notification_type:)
       check_missing_keys(config)
       check_if_form_is_supported(config)
 
-      @form_data = config[:form_data]
+      @form_data = config[:form_data].deep_symbolize_keys
```

Comment:
Just a quick null check here!

```suggestion
      @form_data = config[:form_data]&.deep_symbolize_keys || {}
```